error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/DirectReturnTest.java

```diff
@@ -223,4 +223,19 @@ void replacement() {
             "}")
         .doTest(TestMode.TEXT_MATCH);
   }
+
+  @Test
+  void ignoreClassesAnnotatedWithLombokData() {
+    CompilationTestHelper.newInstance(DirectReturn.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import lombok.Data;",
+            "",
+            "@Data",
+            "class A {",
+            "  private String field;",
+            "}")
+        .expectNoDiagnostics()
```

Comment:
We can drop this, but I think it makes it quite clear though 🤔. Will drop for now.